Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -740,12 +747,16 @@ std::shared_ptr<BankPulseTimes> LoadEventNexus::runLoadNexusLogs(
  * @param nPeriods :: the value in the nperiods log of the run. Number of
  * expected periods
  * @param nexusfilename :: the filename of the run to load
+ * @param status :: will contain any status message. Empty if no problems.
  */
```

Comment:
Can you add an `@returns` (or is it `@return`?) to this doc string now that it returns something.

Reply:
Yes I'll do that